components/events/EventCard.vue

```diff
@@ -9,8 +9,8 @@
     :segment="segment"
     :vertical-layout="verticalLayout"
   >
-    <div class="event-card__description">
-      <slot v-if="$slots.default" />
+    <div v-if="description" class="event-card__description">
```

Comment:
You don't need to change this component, it's already prepare to have the `description`. We did it with slots instead of passing the description just in case that description needed to including something else (links, ...). If we have just a string, we lose that capability

Reply:
Updated in this commit https://github.com/Qiskit/qiskit.org/pull/2965/commits/848504a19790f0d2634aa9adacb76e141951bdde